roles/core/templates/sdcore-5g-values.yaml

```diff
@@ -297,7 +297,8 @@ omec-user-plane:
   nodeSelectors:
     enabled: true
   resources:
-    enabled: false
+    enabled: {{ core.upf.sriov }}
+    privileged: {{ core.upf.sriov }}
```

Comment:
As discussed, let's update the `sriov` variable name to something more generic such that we can "transparently" used it at different places.

Reply:
done